frontend/src/renderer/Svelte.ts

```diff
@@ -1,10 +1,15 @@
 import App from '@root/App.svelte';
 
 const app = new App({
-	target: document.body,
-	// props: {
-	// 	name: 'world'
-	// }
+    target: document.body,
 });
 
+
+// const app = new App({
+// 	target: document.body,
+// 	// props: {
+// 	// 	name: 'world'
+// 	// }
+// });
+
```

Comment:
Can this commented out code be deleted if it is not being used anymore?

Reply:
fixed